sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClient.java

```diff
@@ -727,11 +796,44 @@ public String getDefaultLanguage() {
      *
      * @return A {@link Mono} contains a {@link Response} that contains a {@link AnalyzeSentimentResultCollection}.
      *
-     * @throws NullPointerException if {@code documents} is {@code null}.
+     * @throws NullPointerException if {@code documents} is null.
+     * @throws IllegalArgumentException if {@code documents} is empty.
```

Comment:
Does this exception not apply to the previous API's?

Reply:
inputDocumentsValidation() throws both NullPointerException and IllegalArgumentException. So we should add this exception to here. I think we missed the exception previously. But it should be the correct way to do it.